docs/manual/docs/cmd/spo/folder/folder-copy.md

```diff
@@ -16,6 +16,7 @@ Option|Description
 `-u, --webUrl <webUrl>`|The URL of the site where the folder is located
 `-s, --sourceUrl <sourceUrl>`|Site-relative URL of the folder to copy
 `-t, --targetUrl <targetUrl>`|Server-relative URL where to copy the folder
+`--allowSchemaMismatch`|Ignores any missing fields in the target destination and copies the folder anyway
```

Comment:
_target_ and _destination_ are the same thing. No need to mention them both in one sentence